scripts/benchmark_parser/parse_log.rb

```diff
@@ -66,6 +70,9 @@ def extract_sysbench_results_raw(input_file)
   new_line_count = 0
   File.open(input_file, "r") do |f|
     f.each_line do |line|
+      if line =~ MAXSCALE_COMMIT_REGEX and $maxscale_commit == nil
```

Comment:
Please reuse buildparser logic instead of duplicating code and creating global variable.